content/en/docs/usecases/validation.md

```diff
@@ -78,7 +78,7 @@ Using CUE to generate the schema has several advantages over doing so directly:
 
 - CUE is far less verbose
```

Comment:
We could also add `.` to the end of the first two bullet points for completeness.

Reply:
@myitcv Can do 👍🏻